topics/sequence-analysis/tutorials/quality-control/tutorial.md

```diff
@@ -170,140 +157,304 @@ To estimate sequence quality along all sequences, we now use [FastQC](https://ww
+> #### Small/micro RNA
+> 
+> In small RNA libraries, we typically have a relatively small set of unique, short sequences. Small RNA libraries are not randomly sheared before adding sequencing adapters to their ends: all the reads for specific classes of microRNAs will be identical. It will result in:
+> 
+> - Extremely biased per base sequence content
+> - Extremely narrow distribution of GC content
+> - Very high sequence duplication levels
+> - Abundance of overrepresented sequences
+> - Read-through into adapters
 >
-> > ### {% icon solution %} Solution
-> > In the beginning of sequences, the sequence content per base is not really good and the percentages are not equal. For the GC content, the distribution is slightly shifted on the left, and too high.
-> {: .solution }
-{: .question}
-
+> #### Amplicon
+> 
+> Amplicon libraries are prepared by PCR amplification of a specific target. For example, the V4 hypervariable region of the bacterial 16S rRNA gene. All reads from this type of library are expected to be nearly identical. It will result in:
+>
+> - Extremely biased per base sequence content
+> - Extremely narrow distribution of GC content
+> - Very high sequence duplication levels
+> - Abundance of overrepresented sequences
+>
+> #### Bisulfite or Methylation sequencing
+>
+> With Bisulfite or methylation sequencing, the majority of the cytosine (C) bases are converted to thymine (T). It will result in:
+>
+> - Biased per base sequence content
+> - Biased per sequence GC content
+>
+> #### Adapter dimer contamination
+>
+> Any library type may contain a very small percentage of adapter dimer (i.e. no insert) fragments. They are more likely to be found in amplicon libraries constructed entirely by PCR (by formation of PCR primer-dimers) than in DNA-Seq or RNA-Seq libraries constructed by adapter ligation. If a sufficient fraction of the library is adapter dimer it will become noticeable in the FastQC report:
+>
+> - Drop in per base sequence quality after base 60
+> - Possible bi-modal distribution of per sequence quality scores
+> - Distinct pattern observed in per bases sequence content up to base 60
+> - Spike in per sequence GC content
+> - Overrepresented sequence matching adapter
+> - Adapter content > 0% starting at base 1
+>
+{: .details}
 
 # Filter and Trim
 
-We will filter and trim the sequences based on the information provided by the quality graphs, the quality of the sequences drops at the end of the sequences. This could cause bias in downstream analyses with these potentially incorrectly called nucleotides.
-
-Sequences must be treated to reduce bias in downstream analyis. In general, quality treatments include:
+The quality of the sequences drops at the end of the sequences. This could cause bias in downstream analyses with these potentially incorrectly called nucleotides. Sequences must be treated to reduce bias in downstream analyis. In general, quality treatments include:
```

Comment:
```suggestion
The quality of the sequences drops at the end of the sequences. This could cause bias in downstream analyses with these potentially incorrectly called nucleotides. Sequences must be treated to reduce bias in downstream analysis. In general, quality treatments include:
```